common/config/rush/command-line.json

```diff
@@ -164,65 +158,36 @@
         "enableParallelism": true,
         "ignoreMissingScript": true
       },
-  
+
       {
         "commandKind": "bulk",
         "name": "generate-doc",
         "summary": "Run typedoc on projects and generate documentation",
         "enableParallelism": true,
         "ignoreMissingScript": true
       },
-  
+
       // Global Commands
-      {
-        "commandKind": "global",
-        "name": "list-files",
```

Comment:
I guess these are copied from somewhere but actually doesn't work at all ?

Reply:
yeah looks copied from azsdkjs repo. It's not used anywhere as fare as I know.